Runtime/AvatarCreator/Scripts/UI/Elements/AccountCreationElement.cs

```diff
@@ -20,27 +20,29 @@ public class AccountCreationElement : MonoBehaviour
         // Event invoked when the "Continue Without Signup" button is clicked.
         public UnityEvent OnContinueWithoutSignup;
 
-        private void Update()
-        {
-            var email = emailField.text;
-            sendEmailButton.interactable = !string.IsNullOrEmpty(email) && ValidatorUtil.IsValidEmail(email);
-        }
-
         private void OnEnable()
         {
+            emailField.onValueChanged.AddListener(OnEmailChanged);
```

Comment:
Only thing is I think we should add this extra check just to be safe that when it is reenabled it will be non-interactable if empty. 
```suggestion
            var email = emailField.text;
            sendEmailButton.interactable = !string.IsNullOrEmpty(email) && ValidatorUtil.IsValidEmail(email);
            emailField.onValueChanged.AddListener(OnEmailChanged);
```

Reply:
I made the button not interractable at the beginning. That should cover the cases